Packs/OpenCVE/Integrations/OpenCVE/test_data/CVE-2021-26418.json

```diff
@@ -1 +1,132 @@
+                        "name": "N/A",
+                        "tags": [
+                            "Patch",
+                            "Vendor Advisory"
+                        ],
+                        "refsource": "N/A"
+                    }
+                ]
+            },
+            "data_format": "MITRE",
+            "description": {
+                "description_data": [
+                    {
+                        "lang": "en",
+                        "value": "Microsoft SharePoint Server Spoofing Vulnerability"
+                    }
+                ]
+            },
+            "problemtype": {
+                "problemtype_data": [
+                    {
+                        "description": [
+                            {
+                                "lang": "en",
+                                "value": "NVD-CWE-noinfo"
+                            }
+                        ]
+                    }
+                ]
+            },
+            "data_version": "4.0",
+            "CVE_data_meta": {
+                "ID": "CVE-2021-26418",
+                "ASSIGNER": "secure@microsoft.com"
+            }
+        },
+        "impact": {
+            "baseMetricV2": {
+                "cvssV2": {
+                    "version": "2.0",
+                    "baseScore": 5.8,
+                    "accessVector": "NETWORK",
+                    "vectorString": "AV:N/AC:M/Au:N/C:P/I:P/A:N",
+                    "authentication": "NONE",
+                    "integrityImpact": "PARTIAL",
+                    "accessComplexity": "MEDIUM",
+                    "availabilityImpact": "NONE",
+                    "confidentialityImpact": "PARTIAL"
+                },
+                "severity": "MEDIUM",
+                "acInsufInfo": false,
+                "impactScore": 4.9,
+                "obtainAllPrivilege": false,
+                "exploitabilityScore": 8.6,
+                "obtainUserPrivilege": false,
+                "obtainOtherPrivilege": false,
+                "userInteractionRequired": true
+            },
+            "baseMetricV3": {
+                "cvssV3": {
+                    "scope": "UNCHANGED",
+                    "version": "3.1",
+                    "baseScore": 4.6,
+                    "attackVector": "NETWORK",
+                    "baseSeverity": "MEDIUM",
+                    "vectorString": "CVSS:3.1/AV:N/AC:L/PR:L/UI:R/S:U/C:L/I:L/A:N",
+                    "integrityImpact": "LOW",
+                    "userInteraction": "REQUIRED",
+                    "attackComplexity": "LOW",
+                    "availabilityImpact": "NONE",
+                    "privilegesRequired": "LOW",
+                    "confidentialityImpact": "LOW"
+                },
+                "impactScore": 2.5,
+                "exploitabilityScore": 2.1
+            }
+        },
+        "publishedDate": "2021-05-11T19:15Z",
+        "configurations": [{
```

Comment:
prettified json, and changed it to the new API response format, the configuration from a dict to list

Reply:
Is it a SaaS service?
Can't some customers be on the previous API version?
In that case, you'll need to add a `server_version` param.

Reply:
this is just for Unit testing, and i'm doing backward compatibility check in the code. this API is in beta mode, so it's subject to change. but it only has a single version.